include/integratorxx/atomic_densities/SlaterAtomicShell.hpp

```diff
@@ -0,0 +1,102 @@
+#pragma once
+#include <vector>
+#include <cmath>
+
+class SlaterTypeAtomicShell {
+  /// Angular momentum
+  int angular_momentum;
+  /// Exponents
+  std::vector<double> exponents;
+  /// Principal quantum numbers
+  std::vector<int> quantum_numbers;
```

Comment:
Since these are all private members, we should (at the very lest) add getters, e.g.
```
auto* quantum_numbers_data() const { return quantum_numbers.data(); }
auto quantum_number(size_t i) const { return quantum_numbers[i]; }
```
etc. For `orbital_coefficients`, this should follow matrix indexing.

Reply:
done